js/index.d.ts

```diff
@@ -0,0 +1,87 @@
+/*
+  Typings for @sentry/cli
+*/
+declare module '@sentry/cli' {
+  export interface SentryCliOptions {
+    url?: string;
+    authToken?: string;
+    apiKey?: string;
+    dsn?: string;
+    org?: string;
+    project?: string;
+    vscRemote?: string;
+    silent: boolean;
```

Comment:
`silent` can be marked optional, as we only change the behavior if it's `true`. Any falsy value will work for disabling it.
```suggestion
    silent?: boolean;
```